content/applications/general/calendars/outlook/outlook_calendar.rst

```diff
@@ -1,103 +1,149 @@
-=======================================
-Synchronization with Outlook's Calendar
-=======================================
+======================================
+Synchronize Outlook Calendar with Odoo
+======================================
 
-Synchronizing your **main** Outlook Calendar with Odoo is useful to keep track of your tasks and
-appointments, and across all related applications.
+Synchronizing a user's **main** Outlook Calendar with Odoo is useful for keeping track of their
+tasks and appointments across all related applications.
 
-Register your application with the Microsoft identity platform
-==============================================================
+Register the application with Microsoft Azure
+=============================================
 
-| In order to be able to sync your Outlook Calendar with Odoo's Calendar, a Microsoft Azure account
-  is needed. The creation of an account is free for users that have never tried or paid for Azure.
-  For more information, `click here <https://azure.microsoft.com/en-us/free/?WT.mc_id=A261C142F>`_.
-| Following, `Set up an Azure AD Tenant <https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-create-new-tenant>`_,
-  also called environment, which is a representation of an organization, to manage and register apps.
-| Ultimately, `Register an Application <https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_,
-  choosing the appropriate supported account type.
+To sync the Outlook Calendar with Odoo's Calendar, a Microsoft Azure account is needed. Creating an
+account is free for users who have never tried or paid for Azure. For more information, `click here
+<https://azure.microsoft.com/en-us/free/?WT.mc_id=A261C142F>`_.
 
-.. note::
-   When configuring your platform settings, choose *Web*.
+Refer to `Microsoft's documentation
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-create-new-tenant>`_
+on how to set up an Azure AD Tenant (also called an *environment*), which is a representation of an
+organization to manage and register apps.
+
+Then, `Register an Application
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_,
+choosing the appropriate supported account type. Users who wish to connect their Outlook calendar
+to Odoo should select the :guilabel:`Accounts in any organizational directory (Any Azure AD
+directory - Multitenant and personal Microsoft accounts (e.g. Skype, Xbox)` option for
+:guilabel:`Supported account types`.
 
-   .. image:: outlook_calendar/platform_configurations.png
-      :align: center
-      :alt: Adding a platform in Microsoft Azure for Outlook calendar sync in Odoo
+
+When configuring the platform settings, choose :guilabel:`Web`.
+
+.. image:: outlook_calendar/platform_configurations.png
```

Comment:
Media filenames should be separated with [hypens](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files), not underscores.